src/main/java/Duke.java

```diff
@@ -1,10 +1,239 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    /** Constants used for displaying messages */
+    private static final String BORDER = "____________________________________________________________";
+    private static final String NEWLINE = System.lineSeparator();
+    public static final String LOGO =
```

Comment:
Well done in using all uppercase and underscore to separate words in constant names